staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -224,6 +275,11 @@ func (o *AutoscaleOptions) Run() error {
 		if err := o.handleHPA(hpaV2); err != nil {
 			klog.V(1).Infof("Encountered an error with the v2 HorizontalPodAutoscaler: %v. "+
 				"Falling back to try the v1 HorizontalPodAutoscaler", err)
+			// check if the HPA can be created using v1 API.
+			if !o.canCreateHPAV1() {
```

Comment:
Here we need to determine whether it can be created in V1, because there are different between v1 and v2.